### PR TITLE
Go back

### PR DESCRIPTION
**Pull request by nicosampler**

### CODE REVIEW

File: pages_/shop-form.tsx [hunk 1]
```diff
@@ -495,6 +496,17 @@ const EditShop = ({ isLoggedIn, shop }: Props) => {
...
+              block
+              href="/"
+              type="button"
+              disabled={isSubmitting}
+              variant="danger"
+              size="lg"
+              className="mt-4"
+            >
+              {t('common:cancel')}
+            </Button>
```
Comment (ramirotw) [inline pages_/shop-form.tsx:509]: This should be wrapped with a Link as other parts of the app. It should also link to /my-shop

File: pages_/scan.tsx [hunk 2]
```diff
@@ -53,6 +53,14 @@ const Scan = () => {
...
   };
 
+  useEffect(() => {
+    const interval = setInterval(() => {
+      Router.push('/');
+    }, 10_000);
+
+    return () => clearInterval(interval);
+  }, []);
+
```
Comment (ramirotw) [inline pages_/scan.tsx:63]: I believe this is confusing, if we are not going to ship the scanner during the pilot it should be hidden from the home screen.

File: pages_/shop-form.tsx (hunk 1 above)
Comment (nicosampler) [reply pages_/shop-form.tsx:509]: This is a form, I think having a Cancel button is the best approach. But we can have both, to be consistent with other screens.

File: pages_/scan.tsx (hunk 2 above)
Comment (nicosampler) [reply pages_/scan.tsx:63]: Almost all apps I have tried, they have a timeout for the cam scan, I think is a common use case.

File: pages_/shop-form.tsx (hunk 1 above)
Comment (app-cauda) [reply pages_/shop-form.tsx:509]: I mean, the button is ok, but it needs to be wrapped with a Link so it enables client side routing

File: pages_/scan.tsx (hunk 2 above)
Comment (ramirotw) [reply pages_/scan.tsx:63]: Why a timeout? Like I only have 10 seconds to scan the QR code? If I have a low-budget cellphone that takes a lot of time to focus what happens?

File: pages_/shop-form.tsx (hunk 1 above)
Comment (nicosampler) [reply pages_/shop-form.tsx]: done

File: pages_/scan.tsx (hunk 2 above)
Comment (nicosampler) [reply pages_/scan.tsx]: timeout removed and cancel button added instead.

**Pull request merged**